src/main/resources/brp-api-gezag.yaml

```diff
@@ -0,0 +1,192 @@
+
+    Het antwoord van de Gezagsmodule komt in de vorm van een lijst met daarin de BSN minderjarige, Soort
+    Gezagt tussen relatie met meerderjarige en BSN meerderjarige
+  version: "1.0"
+  contact:
+    url: https://brp-api.github.io/Haal-Centraal-BRP-bevragen
+  license:
+    name: European Union Public License, version 1.2 (EUPL-1.2)
+    url: https://eupl.eu/1.2/nl/
+servers:
+  - description: |
+      Proef omgeving
+    url: https://proefomgeving.haalcentraal.nl/haalcentraal/api/brp
+tags:
+  - name: Personen
+externalDocs:
+  description: BRP API Personen website
+  url: https://brp-api.github.io/Haal-Centraal-BRP-bevragen
+paths:
+  /api/v1/opvragenBevoegdheidTotGezag:
+    post:
+      tags:
+        - GezagsmoduleInterface
+      description: >-
+        De volgende situaties kunnen (gelijktijdig) voorkomen:
+
+        - Bij aanwezigheid van juridische kinderen wordt het gezag van deze persoon over diens kinderen teruggegeven. Het kan zijn dat omwille van het reduceren van BRP bevragingen de leeftijd van de kinderen uit de lijst met kinderen van de ouder wordt gebruikt om kinderen die al de leeftijd van 18 hebben bereikt niet te bevragen. dit heeft wel impact op de als minderjarige overleden kinderen; daar wordt dan geen gezagsuitspraak gedaan.
+
+        - Is de persoon minderjarig dan wordt het gezag van de ouder(s) over deze persoon teruggegeven.
+      operationId: opvragenBevoegdheidTotGezag
+      parameters:
+        - name: OIN
+          in: header
+          required: true
+          schema:
+            type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/GezagRequest"
+        required: true
+      responses:
+        "200":
+          description: OK
+          headers:
+            TransactionID:
+              description: UUID gegenereerd voor de transactie
+              style: simple
+              schema:
+                type: string
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/GezagResponse"
+        "503":
+          description: Gegevensbron niet beschikbaar
+          content:
+            application/json:
+              schema:
+                type: string
+components:
+  schemas:
+    GezagRequest:
+      type: object
+      properties:
+        burgerservicenummer:
+          type: array
+          items:
+            $ref: "#/components/schemas/Burgerservicenummer"
+          minItems: 1
+          maxItems: 20
+    GezagResponse:
+      type: object
+      properties:
+        personen:
+          type: array
+          items:
+            $ref: "#/components/schemas/Persoon"
+    Burgerservicenummer:
+      type: string
+      pattern: ^[0-9]{9}$
+      example: "555555021"
+    AbstractGezagsrelatie:
+      type: object
+      required:
+        - type
+      properties:
+        type:
+          type: string
+        toelichting:
```

Comment:
Zie comment https://github.com/BRP-API/brp-api-gezag/issues/2#issuecomment-2235838685 van @CathyDingemanse> Zij stelt voor om alleen toelichting te geven wanneer gezag niet kan worden bepaald. Veld toelichting moet dan niet in AbstractGezagsrelatie zitten, maar in GezagNietTeBepalen.